extensions-core/datasketches/src/test/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchAggregatorFactoryTest.java

```diff
@@ -0,0 +1,290 @@
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.datasketches.hll;
+
+import com.yahoo.sketches.hll.HllSketch;
+import com.yahoo.sketches.hll.TgtHllType;
+import org.apache.druid.query.aggregation.Aggregator;
+import org.apache.druid.query.aggregation.AggregatorFactory;
+import org.apache.druid.query.aggregation.BufferAggregator;
+import org.apache.druid.segment.ColumnSelectorFactory;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.annotation.Nullable;
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static junit.framework.TestCase.assertNull;
+import static org.easymock.EasyMock.expect;
+import static org.easymock.EasyMock.mock;
+import static org.easymock.EasyMock.replay;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertTrue;
+
+public class HllSketchAggregatorFactoryTest
+{
+  private static final String NAME = "name";
+  private static final String FIELD_NAME = "fieldName";
+  private static final int LG_K = HllSketchAggregatorFactory.DEFAULT_LG_K;
+  private static final String TGT_HLL_TYPE = TgtHllType.HLL_4.name();
+  private static final boolean ROUND = true;
+  private static final double ESTIMATE = Math.PI;
+
+  private TestHllSketchAggregatorFactory target;
+
+  @Before
+  public void setUp()
+  {
+    target = new TestHllSketchAggregatorFactory(NAME, FIELD_NAME, LG_K, TGT_HLL_TYPE, ROUND);
+  }
+
+  @Test
+  public void testIsRound()
+  {
+    assertEquals(ROUND, target.isRound());
+  }
+
+  @Test
+  public void testGetRequiredColumns()
+  {
+    List<AggregatorFactory> aggregatorFactories = target.getRequiredColumns();
+    assertEquals(1, aggregatorFactories.size());
+    HllSketchAggregatorFactory aggregatorFactory = (HllSketchAggregatorFactory) aggregatorFactories.get(0);
+    assertEquals(FIELD_NAME, aggregatorFactory.getName());
+    assertEquals(FIELD_NAME, aggregatorFactory.getFieldName());
+    assertEquals(LG_K, aggregatorFactory.getLgK());
+    assertEquals(TGT_HLL_TYPE, aggregatorFactory.getTgtHllType());
+    assertEquals(ROUND, aggregatorFactory.isRound());
+  }
+
+  @Test
+  public void testFinalizeComputation_null()
```

Comment:
Minor: Most of our test files use camel case instead of underscores for things like this.